.travis.yml

```diff
@@ -12,5 +12,6 @@ before_install:
 install:
   - "pip install xcffib"
   - "pip install ."
+  - "if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then python cairocffi/ffi_build.py; fi"
```

Comment:
Right, it is the difference between local files and installed files. Using `--pyargs` works, see the current version, but I also needed to cd out of the directory so it didn't pick up the uninstalled source again.